x-pack/plugins/lens/public/embeddable/embeddable_component.tsx

```diff
@@ -52,7 +52,7 @@ export type TypedLensByValueInput = Omit<LensByValueInput, 'attributes'> & {
 };
 
 export type EmbeddableComponentProps = (TypedLensByValueInput | LensByReferenceInput) & {
-  withActions?: boolean;
+  withActions?: boolean | Action[];
```

Comment:
It seems a bit weird to re-use the existing prop for this - what about splitting out `extraActions` on this level as well?

Reply:
i find using existing prop as a client  to be honest cleaner. I mean now if use has to pass extra actions, they just passed it as an array. 

Adding an extra prop means, we will have to `withActions` true and then pass `extraActions` prop. 

Happy to amend it, if you feel strong about it.

Reply:
It's definitely a nit, I'm just a bit concerned people will be confused (I was initially). Maybe we can further disambiguate and call it `withDefaultActions` and `extraActions`? That what you can also _only_ show your special extra actions and hide the default ones.

Reply:
done